pacai/ui/graphicsConstants.py

```diff
@@ -0,0 +1,60 @@
+from pacai.ui import graphicsUtils
+
+DEFAULT_GRID_SIZE = 30.0
+INFO_PANE_HEIGHT = 35
+BACKGROUND_COLOR = graphicsUtils.formatColor(0, 0, 0)
+WALL_COLOR = graphicsUtils.formatColor(0.0 / 255.0, 51.0 / 255.0, 255.0 / 255.0)
+INFO_PANE_COLOR = graphicsUtils.formatColor(0.4, 0.4, 0)
+SCORE_COLOR = graphicsUtils.formatColor(0.9, 0.9, 0.9)
+PACMAN_OUTLINE_WIDTH = 2
+PACMAN_CAPTURE_OUTLINE_WIDTH = 4
+WHITE = graphicsUtils.formatColor(1.0, 1.0, 1.0)
+BLACK = graphicsUtils.formatColor(0.0, 0.0, 0.0)
+
+
+GHOST_COLORS = [
+    graphicsUtils.formatColor(0.9, 0.0, 0.0),  # Red
+    graphicsUtils.formatColor(0.0, 0.3, 0.9),  # Blue
+    graphicsUtils.formatColor(0.98, 0.41, 0.07),  # Orange
+    graphicsUtils.formatColor(0.1, 0.75, 0.7),  # Green
+    graphicsUtils.formatColor(1.0, 0.6, 0.0),  # Yellow
+    graphicsUtils.formatColor(0.4, 0.13, 0.91),  # Purple
+]
+
+TEAM_COLORS = GHOST_COLORS[:2]
+
+GHOST_SHAPE = [
+    (0, 0.3),
+    (0.25, 0.75),
+    (0.5, 0.3),
+    (0.75, 0.75),
+    (0.75, -0.5),
+    (0.5, -0.75),
+    (-0.5, -0.75),
+    (-0.75, -0.5),
+    (-0.75, 0.75),
+    (-0.5, 0.3),
+    (-0.25, 0.75)
+]
+GHOST_SIZE = 0.65
+SCARED_COLOR = graphicsUtils.formatColor(1, 1, 1)
+
+GHOST_VEC_COLORS = list(map(graphicsUtils.colorToVector, GHOST_COLORS))
+
+PACMAN_COLOR = graphicsUtils.formatColor(255.0 / 255.0, 255.0 / 255.0, 61.0 / 255)
```

Comment:
Make sure all floating point numbers are of the same format of a leading zero if it is below 1 (i.e. 0.2 not .2) and there is always a ending zero for integer values (255.0 not 255). It is important that integer values are not changed to floating point unless the problem warrants it. For example counting number of iterations should be integer not floating, but dividing without rounding is done with floating.

Reply:
https://github.com/linqs/pacman/blob/7ea06f7529810bad68f21b6ff3e482c228f86ed3/pacai/ui/graphicsUtils.py#L17-L18

Some calls to this method in graphicsUtil.py pass in floating point ones and zeroes but since the method casts its arguments to integers it doesn't matter if 1 is passed as '1' or '1.0' and similarly '0' or '0.0' so i will be changing the numbers in the two scenarios to their integer versions.